xapian-applications/omega/clickmodel/simplifiedDBN.h

```diff
@@ -0,0 +1,61 @@
+/** @file simplifiedDBN.h
+ * @brief SimplifiedDBN class - the Simplified DBN click model.
+ */
+/* Copyright (C) 2017 Vivek Pal
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#include <array>
+#include <map>
+#include <string>
+#include <vector>
+
+using namespace std;
+
+/** 
+ * SimplifiedDBN class implementing the SDBN click model.
+ *
+ * For more information of DBN click model, see the following paper:
+ *
+ * Olivier Chapelle and Ya Zhang. 2009. A dynamic bayesian network click
+ * model for web search ranking. In Proceedings of the 18th international
+ * conference on World wide web (WWW '09). 
+ */
+class SimplifiedDBN {
+	///
+	map<string, map<string, map<char, double>>> urlRelevances;
```

Comment:
> It's not clear to me what these mean, and it's overkill to use a map for two fixed values.

`'a'` and `'s'` refer to the model parameters: attractiveness probabilities and satisfaction probabilities as mentioned in the paper.

> I'd suggest defining constants instead of using single characters, for example (except give them meaningful names instead of using A and S):

Yes, thanks for the suggestion of using `enum`. I've made the required changes.